src/transaction/log_tran_table.c

```diff
@@ -6140,6 +6142,7 @@ log_tdes::on_sysop_start ()
       LSA_SET_NULL (&tail_lsa);
       LSA_SET_NULL (&undo_nxlsa);
       LSA_SET_NULL (&tail_topresult_lsa);
+      LSA_SET_NULL (&commit_abort_lsa);
```

Comment:
I don't think this is necessary. System transactions never commit and never make use of `commit_abort_lsa`.
```suggestion
```

Reply:
Replace with an assert then?

Reply:
Replacing with assert is a valid option.